server/src/controllers/projects/getProjectsController.ts

```diff
@@ -0,0 +1,15 @@
+import { Response, Request } from 'express';
+import { getProjectsQuery } from '../../database';
+
+const getProjects = async (req: Request, res: Response) => {
+  const { userId } = req.body;
+  try {
+    const { rows } = await getProjectsQuery(userId);
+    if (!rows) return res.status(204).json({ message: 'No data found' });
```

Comment:
here, you should check the length of array rows, or take rowCount from your query,
empty array will return true value always!